analyzers/nodejs/nodejs_test.go

```diff
@@ -13,6 +13,46 @@ import (
 	"github.com/fossas/fossa-cli/pkg"
 )
 
+// testNDepsTransitiveImports checks that Analyze() returns the correct transitive deps
+// by asserting on transitive dependencies
```

Comment:
[nit] I think this could be more terse. Something along the lines of `verifies the imports of every transitive dep returned from Analyze`. or something to that effect. Right now I'm unsure what you mean by 'correct transitive deps`. What makes them correct?

Reply:
Previously we were returning every transitive dep, we just weren't returning them in the correct order or under the correct dependencies. Now we are asserting that the graph is structured correctly, ensuring that each package has the correct list of imports.